db/seeds.rb

```diff
@@ -22,7 +22,7 @@
 referral_template = EmailTemplate.find_or_initialize_by(name: 'Referral Invitation')
 referral_template.update(
   subject: 'Invite Your Friends!',
-  body: 'Hi there, <br>Invite your friends using this link: <a href="http://localhost:3000/refer_a_friend">Click here</a>.'
+  body: 'Hi there, <br>Invite your friends using this link: <a href="PLACEHOLDER_LINK">Click here</a>.'
```

Comment:
We need to change this. Maybe not in this PR, but the link shouldn't be injected by seeding db. There should some mechanism to generate the link and let users attach it into the template.